docs/target-browser/README.md

```diff
@@ -0,0 +1,89 @@
+# TVK Target Browser Plugin
+  
+  **tvk-target-browser** plugin queries content of mounted target location to get details of backup, backupPlan and
+metadata details of backup via HTTP/HTTPS calls to target-browser server.
+Plugin currently supports GET operation on target-browser's `/backupplan`, `/backup` and `/metadata` API.
+ 
+## Pre-requisites:
+  
+  1. krew - kubectl-plugin manager. Install from [here](https://krew.sigs.k8s.io/docs/user-guide/setup/install/)
+  2. kubectl - kubernetes command-line tool. Install from [here](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+  3. TVK's web-backend service and target-browser server should be up and running.
```

Comment:
instead of target-browser server we should say browsing flag should be enabled for the target you want to query

Reply:
Sure, updated accordingly.